src/ripper/ripcddialog.cpp

```diff
@@ -128,6 +129,9 @@ RipCDDialog::RipCDDialog(QWidget* parent)
       break;
     }
   }
+
+  // Start thread for polling
```

Comment:
This isn't really a thread.

Reply:
true, it's not anymore. I'll fix the comment